### DISCUSSION TITLE
Different follower numbers

### DISCUSSION THREAD

**Discussion by Glowstudent777** [community/community#57550] [Mobile] [answered]
Hi, I noticed this yesterday and waited till today to make sure it wasn't just delayed. According to GitHub.com I have 116 followers and according to the GitHub mobile app I have 110.

![Screenshot_20230608-093818.png](https://github.com/community/community/assets/55334764/6a3a1f6e-7397-4dc3-a2a7-c4d082ed69a1)

![Screenshot_20230608-093833.png](https://github.com/community/community/assets/55334764/501bc76c-4592-461f-906d-520ef9ef97f4)

**Comment by HenSquared** [comment, marked as answer]
Hi @Glowstudent777 , we have encountered this issue before and have an Issue already open tracking it. I will link this feedback to that issue to help us get it fixed. Thanks a lot for your feedback!
